dev/merge_spark_pr.py

```diff
@@ -403,6 +403,19 @@ def choose_jira_assignee(issue, asf_jira):
             print("Error assigning JIRA, try again (or leave blank and fix manually)")
 
 
+def assign_issue(client: jira.client.JIRA, issue: int, assignee: str) -> bool:
```

Comment:
Hi, this breaks the environment where `jira` is not installed.

Reply:
hmm, AFAIK the merge script required `jira` before this PR, as we need to update the JIRA ticket status.

Reply:
It still worked, just skipped to update the JIRA ticket status.
It's fine if it's a required dependency, though.

Reply:
I apologize for any inconvenience caused. I didn't notice jira is optional. I will make a follow-up to restore this.